### ISSUE TITLE
Level 7 vocab word

### ISSUE THREAD

**Issue by Jacquelyn-Rudis** [YMCACOLUMBUS/activescience-issues#46]
Screen after "nice job you've earned __" points (the one that gives the vocab word "conclusion"): "We'll be making a conclusion today on the data you've collected in the past 4 days" - since they don't do Active Science every day should this say "past 4 levels"? It may be too wordy to say "past 4 times you have done Active Science"

**Comment by TAGraves** [comment]
I will change it to past 4 levels.

**Comment by TAGraves** [comment]
Fixed in commit d5ffa81c6f53a5dd027b2b9c0af6f37f64937fd2, will be available tomorrow.
